pkg/cmd/openshift/openshift.go

```diff
@@ -96,6 +96,8 @@ func NewCommandOpenShift(name string) *cobra.Command {
 	switch name {
 	case "openshift":
 		product = "OpenShift"
+	case "atomic-enterprise":
+		product = "Atomic"
```

Comment:
On 7.8.2015 20:23, Clayton Coleman wrote:
> In pkg/cmd/openshift/openshift.go <https://github.com/openshift/origin/pull/3953#discussion_r36546501>:
>
>> @@ -96,6 +96,8 @@ func NewCommandOpenShift(name string) *cobra.Command {
>>  	switch name {
>>  	case "openshift":
>>  		product = "OpenShift"
>> +	case "atomic-enterprise":
>> +		product = "Atomic"
>
> should be Atomic Enterprise Platform
>
> —
> Reply to this email directly or view it on GitHub <https://github.com/openshift/origin/pull/3953/files#r36546501>.
>
Ahh, the naming again. This variable is expanded in help template to

- `$product Application Platform`
- `The $product distribution of Kubernetes`

`Atomic server` has been PM approved. This is kind of similar. But as for product name, you're right. `Atomic Enterprise 
Platform` is its full name. Would it be OK to use two variables: `product` and `distribution`? The first would expand to:

- `Origin Application Platform`
- `OpenShift Application Platform`
- `Atomic Enterprise Platform`

And the `distribution` to:

- `Origin distribution of Kubernetes`
- `OpenShift distribution of Kubernetes`
- `Atomic distribution of Kubernetes`

Let's have some utility getters for both of them based on real executable
path checking. WDYT?